### PR TITLE
WIP: much documentation updates

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, retest this please

**Comment by mr-c** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Is the Copyright year up to date?

File: doc/dev/scripts-and-sandbox.rst [hunk 1]
```diff
@@ -58,7 +58,10 @@ All scripts in scripts/ should have the following options, if they could apply:
 * ``--version`` - should always apply
 * ``--help`` - should always apply
 * ``--force`` - override any sanity checks that may prevent the script from running
-* ``--loadtable`` and ``--savetable`` - where appropriate (see khmer_args.py)
+* ``--loadgraph`` and ``--savesave`` - where appropriate (see khmer_args.py)
```
Comment (bocajnotnef) [inline doc/dev/scripts-and-sandbox.rst]: "savesave"

File: doc/dev/scripts-and-sandbox.rst [hunk 2]
```diff
@@ -58,7 +58,10 @@ All scripts in scripts/ should have the following options, if they could apply:
...
+
+If an option is of type ``type=argparse.FileType('w')`` then you need to also
+specify a ``metavar`` for the documentation and help formatting.
```
Comment (bocajnotnef) [inline doc/dev/scripts-and-sandbox.rst]: example metavar?

File: doc/user/guide.rst [hunk 3]
```diff
@@ -237,10 +237,10 @@ For a cleverer approach that we will someday implement, read `the
 Beachcomber's Dilemma
 <http://ivory.idyll.org/blog/beachcombers-dilemma.html>`__.
 
-Validating and comparing assemblies
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. Validating and comparing assemblies
+.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-More here soon :).
+.. More here soon :).
```
Comment (bocajnotnef) [inline doc/user/guide.rst]: -1 for smiley

**Comment by bocajnotnef** [comment]
other than the above LGTM

File: doc/user/guide.rst (hunk 3 above)
Comment (mr-c) [inline doc/user/guide.rst]: Which is why I commented it out :-)

Comment (bocajnotnef) [inline doc/user/guide.rst]: ohh, that's what that does...

I was thinking that's a toctree type thing.

**Comment by bocajnotnef** [comment]
Are you fixing the copyright message in https://github.com/dib-lab/khmer/blob/master/doc/dev/scripts-and-sandbox.rst or am I?

**Comment by mr-c** [comment]
Jenkins, retest this please

**Comment by mr-c** [comment]
@bocajnotnef ready for review & merge

**Comment by bocajnotnef** [comment]
Merged #1222

**Pull request merged**